src/otx/core/metrics/ap_3d.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Module for OTX  metric used for 3D object detection tasks."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+import numpy as np
+from torch import Tensor
+from torchmetrics import Metric
+
+from otx.core.metrics.kitti_3d_eval import get_coco_eval_result
+
+if TYPE_CHECKING:
+    from otx.core.types.label import LabelInfo
+
+
+class KittiMetric(Metric):
+    """Computes the f-measure (also known as F1-score) for a resultset.
```

Comment:
This docstring is about f-measure, not average precision. Please update it.

Reply:
Done